src/gitea-proxy/test/Makefile

```
@@ -0,0 +1,76 @@
+	$(COMPOSE) build
+
+cases:
+	@$(HURL) --test $(HURL_ARGS) $(HURL_FILES)
+
+public:
+	@$(HURL) --test $(HURL_ARGS) public.hurl
+
+trusted:
+	@$(HURL) --test $(HURL_ARGS) trusted.hurl
+
+wait:
```

checkmake (0.3.2)

[warning] Target body for "wait" exceeds allowed length of 5 lines (8).

(maxbodylength)